test_sets/policies_smoke.txt

```diff
@@ -0,0 +1,6 @@
+./test_scripts/Polices/build_options/001_ATF_P_TC_SDL_Build_Flag_DEXTENDED_POLICY_PROPRIETARY.lua
+./test_scripts/Polices/build_options/ATF_PTU_Trigger_kilometers.lua
```

Comment:
@istoimenova, yes sure
But scripts you mentioned have an issues and was excluded from CI.
Only successful scripts can be selected for smoke.
Whole list can be found for example here: http://opensdl-jenkins.luxoft.com:8080/job/Policies_Nightly_Status/job/ATF_Policies_Nightly_PROPRIETARY/lastSuccessfulBuild/artifact/atf_report.html